lib/plugins/create/templates/azure-nodejs-service-bus-queue/src/handlers/hello.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports.sayHello = async function(context, req) {
+  context.log('JavaScript HTTP trigger function processed a request.');
```

Comment:
Does it make sense to include http handlers in the template for `service-bus-queue` (or any other type besides HTTP for that matter? I understand that people could use it this way, but it might be a bit confusing to see in a template

Reply:
the http trigger is just there to demonstrate how to have 2 function in the yaml.  Not sure if this what you're asking